lhotse/dataset/speech_recognition.py

```diff
@@ -57,3 +58,122 @@ def __getitem__(self, idx: int) -> Dict[str, torch.Tensor]:
 
     def __len__(self) -> int:
         return len(self.cut_ids)
+
```

Comment:
Thanks...
What about collation?  How would one use this from PyTorch (I am not super familiar with such things).

Reply:
We'll have to provide a custom collate_fn to the DataLoader that converts multiple `Fsa`s into `FsaVec` if I understood K2 approach correctly (or a list of `Fsa`? not sure what's more convenient). Fangjun helped me with K2 installation so I'll play around with that today.